### PR TITLE
gradle wrapper content

### PR DESCRIPTION
**Pull request by DannyS03**
https://github.com/OpenLiberty/guide-gradle-intro/issues/8
https://github.com/OpenLiberty/guide-gradle-intro/pull/55#pullrequestreview-321134616

### CODE REVIEW

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt, would you review/approve the README.adoc change? The updated guide is available at http://lgdev1.fyre.ibm.com:4020/guides/gradle-intro.html

File: README.adoc [hunk 1]
```diff
@@ -75,47 +77,76 @@ is in the `start/src` directory:
...
 
-== Installing Gradle
+== Testing Gradle
 
-If you don't have Gradle Version 3 or higher installed, follow the instructions on the 
-https://gradle.org/install/[Gradle installation page^] for your operating system.
-Configure the environment variables so that you can run the `gradle` command.
+If you do not have Gradle installed, running the Gradle Wrapper will automatically install Gradle. The Gradle 
+Wrapper files are included in the project directory. Learn more about Gradle Wrapper on the 
+https://docs.gradle.org/current/userguide/gradle_wrapper.html[official website^].
```
Comment (Charlotte-Holt) [inline README.adoc]: Change
"Learn more about Gradle Wrapper on the https://docs.gradle.org/current/userguide/gradle_wrapper.html[official website^]."
To
"To learn more about Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle documentation^]."

File: README.adoc [hunk 2]
```diff
@@ -75,47 +77,76 @@ is in the `start/src` directory:
...
 
 Test that Gradle is installed properly by running the following command in a command line:
```
Comment (Charlotte-Holt) [inline README.adoc]: Change to
"Test that Gradle is installed properly by running the following command:"

File: README.adoc (hunk 1 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 2 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

**Comment by gkwan-ibm** [comment]
@Charlotte-Holt Danny addressed your suggestions.

**Pull request merged**